nubilanding/src/components/recargas/recargas.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styles from '../recargas/styles.module.scss';
+import LogoPlay from '../../assets/img_logo_playstation.png';
+import LogoForever from '../../assets/img_logo_forever21.png';
+import LogoTwitch from '../../assets/img_logo_twitch.png';
+import LogoTripAdvisor from '../../assets/img_logo_tripadvisor.png';
+import LogoAirbnb from '../../assets/img_logo_airbnb.png';
+import LogoEbay from '../../assets/img_logo_ebay.png';
+import LogoTiendaMia from '../../assets/img_logo_tiendamia.png';
+import LogoCarters from '../../assets/img_logo_carters.png';
+import LogoOrigin from '../../assets/img_logo_origin.png';
+
+class Recargas extends React.Component {
+    render () {
+        return (
+            <div className={styles.contenedorGeneral}> 
+                <div className={styles.contenedorRecarga}>
+                    <h2 className={styles.titulos}>Recargá tu cuenta PayPal</h2>
+                    <p className= {styles.textoRecargas}>
+                        Con Nubi agregá dinero a tu cuenta PayPal desde tu banco y comprá en todo el mundo sin tajeta de crédito. ¡Si no tenés cuenta PayPal creátela desde Nubi en unos simples pasos! 
+                    </p>
+                    <p className= {styles.textoRecargas}>
+                        El costo de la recarga es de 5% + IVA sobre la tarifa.
+                    </p>
+                    <button className={styles.boton}>Registrate Gratis</button>
+                </div>
+                <div className={styles.contenedorAhorra}>
+                    <h2 className={styles.titulos}>Ahorrá en tus compras online</h2>
+                    <div className={styles.contenedorLogosPF}>
+                        <img className={styles.logo} src= {LogoPlay} alt="Logo Playstation Store"/>
+                        <img className={styles.logo} src= {LogoForever} alt="Logo Forever 21"/>
+                        <img className={styles.logo} src= {LogoTwitch} alt="Logo Twitch"/>
+                        <img  src= {LogoTripAdvisor} alt="Logo Trip Advisor"/>
```

Comment:
¿Estas seguro que al quitar la clase `logo` te sigue funcionando?

Asegúrate que el pseudo selector esté usando la clase como debe ser:

```suggestion
                        <img className={styles.logo} src={LogoTripAdvisor} alt="Logo Trip Advisor"/>
```

Después en el SCSS para seleccionar el último elemento que tenga la clase `.logo` harías algo como:
`.logo:last-child { ... }`

Más sobre [:last-child](https://www.w3schools.com/cssref/sel_last-child.asp)